client/my-sites/people/edit-team-member-form/index.jsx

```diff
@@ -355,4 +383,18 @@ module.exports = protectForm( React.createClass( {
 			</Main>
 		);
 	}
-} ) );
+}
+
+export default connect(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+		const site = getSelectedSite( state );
+
+		return {
+			siteId,
+			siteSlug: getSiteSlug( state, siteId ),
```

Comment:
Since this is the slug of the selected site, would it make more sense to use `getSelectedSiteSlug` instead?